docs/user-guide/timeline.md

```diff
@@ -1,114 +1,90 @@
+<img src="../img/timeline/timeline-animation-buttons.jpg" class="ms-docimage"/>
 
-Clicking the **Play** button, by default, the current time cursor reach the initial time of the ***guide layer*** and the animation starts. You will see the layers' configurations changes on the map and the time progress on the histogram. Following the step sequence, the cursor will shift each time to the next step in a certain time interval, the *frame duration*.
+In order to start the animation the user can click on **Play** button <img src="../img/button/timeline-play-button.jpg" class="ms-docbutton"/>. Once the animation is starts, the layer configuration changes on the map and the time progress on the histogram. Following the sequence of the step, the cursor will shift each time to the next step in a certain time interval, the *frame duration*.
 <br>
-The **Stop** button obviously stops the animation and the current time cursor remains in the last position reached.
+Through the **Stop** button <img src="../img/button/timeline-stop-button.jpg" class="ms-docbutton"/> the user can stop the animation and the current time cursor remains in the last position reached.
 <br>
-The **Step backward** or the **Step forward** buttons allow you to statically change the current time, so by clicking on one of them the cursor changes its position (to the previous or the next step) on the histogram and the date/time values of the control cells will be update accordingly.
+The **Step backward** button <img src="../img/button/timeline-step-backward-button.jpg" class="ms-docbutton"/> and the **Step forward** button <img src="../img/button/timeline-step-forward-button.jpg" class="ms-docbutton"/> allow the user to statically change the current time, so by clicking on one of them the cursor changes its position (to the previous or the next step) on the histogram and the date/time values of the control cells will be update accordingly.
 <br>
-You can pause the animation with the **Pause** button <img src="../img/button/timeline-pause-button.jpg" class="ms-docbutton"/>.
+The user can pause the animation through the **Pause** button <img src="../img/button/timeline-pause-button.jpg" class="ms-docbutton"/>, as follows:
 
 <img src="../img/timeline/timeline-animation.gif" class="ms-docimage"  style="max-width:700px;" />
 
-You can also specify a *time range*. During the animation, the whole range will be shifted step by step along the time axis and in each step the layers' configurations will be consider the total amount of data in that range.
 
-<img src="../img/timeline/timeline-animation-range.gif" class="ms-docimage"  style="max-width:700px;"/>
+!!! Note
+    The user can also specify a *time range*. During the animation, the whole range will be shifted step by step along the time axis and in each step the layer configuration will be consider the total amount of data in that range.
```

Comment:
The user can also specify a *time range*. During the animation, the whole range will be shifted step by step along the time axis and, in each step, the layers in map will show data corresponding to that range of time.
    <img src="../img/timeline/timeline-animation-range.gif" class="ms-docimage"  style="max-width:700px;"/>